### PR TITLE
Code coverage: add tests 

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Signed-off-by: Pierre-Emmanuel Jacquier <pierre-emmanuel.jacquier@epitech.eu>

### CODE REVIEW

File: networks_test.go [hunk 1]
```diff
@@ -393,6 +393,120 @@ func TestListNetworkEmpty(t *testing.T) {
 	}
 }
 
+func TestListNetworkFailure(t *testing.T) {
+	ts := newServer(response{200, `
+{"listNetworksresponse": {
+	"count": 3456,
```
Comment (greut) [inline networks_test.go]: Interesting, does it paginate?

Comment (pierre-emmanuelJ) [reply networks_test.go]: Yes but on the real number of items and not 3456 times

Comment (greut) [reply networks_test.go]: right, `if len(Network) == 0; break`

Comment (pierre-emmanuelJ) [reply networks_test.go]: @greut https://github.com/exoscale/egoscale/issues/81

**Comment by pierre-emmanuelJ** [comment]
@greut https://github.com/exoscale/egoscale/issues/81

**Comment by pierre-emmanuelJ** [comment]
My bad when you make a test it doesn't care of required fields

**Comment by pierre-emmanuelJ** [comment]
@greut errors on some Golang versions

**Pull request merged**